### PR TITLE
SBT opts plus release wdl updates.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -57,6 +57,7 @@ pages:
     - Errors: api/ErrorHandling.md
 - For Developers:
     - How to Build: Building.md
+    - How to Release: Releasing.md
```
Comment (mcovarr) [inline mkdocs.yml]: These instructions are really for Red (or maybe Greater Batch) use only. Do they really belong among these general developer docs?

Comment (kshakir) [reply mkdocs.yml]: I concur. Will try to get consensus on a better spot.

Happy to put them wherever folks will actually find them. I did the release from asking around plus memory. The docs I did find after the release were 1) out of date and 2) still weren't being followed. Ex: When I looked through Homebrew we hadn't done a release in a while.

File: release/release_workflow.wdl [hunk 2]
```diff
@@ -7,49 +7,50 @@ task do_release {
...
-     git pull --rebase
      git merge develop --no-edit
-     
+
      # Make sure tests pass
      sbt update
-     JAVA_OPTS=-XX:MaxMetaspaceSize=1024m sbt test
-     
+     # Compile, with a backup just in case of a "(cwl / Compile / compileIncremental) java.lang.StackOverflowError"
+     # May be related to https://github.com/scala/bug/issues/10911 ?
```
[the remark this answers is not in the document]
Comment (danbills) [reply release/release_workflow.wdl]: I've seen it too,  the solution to this is setting `-Xss:512m`.  Circe just uses a lot of stack

So line above:
`JAVA_OPTS='-XX:MaxMetaspaceSize=1024m -Xss512m' sbt test`

**Comment by kshakir** [comment]
PR redo. This on moves the docs to google and increses sbt jvm stack size to avoid "(cwl / Compile / compileIncremental) java.lang.StackOverflowError".

**Comment by mcovarr** [comment]
still 👍

**Comment by Horneth** [comment]
👍

**Pull request merged**